server/api.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
+	config := p.getConfiguration()
+	if err := config.IsValid(); err != nil {
+		http.Error(w, "This plugin is not configured.", http.StatusNotImplemented)
+		return
+	}
+
+	path := r.URL.Path
+
+	switch {
+	// meeting route with id
+	case strings.Index(path, "/meetings/") == 0:
```

Comment:
2/5 `strings.HasPrefix` would be more intention-revealing

Reply:
Agreed, cleaner as well.